core: adjust styling to meet our standards

kensternberg-authentik opened the pull request:
## Details

The last merge didn't check the styling, and `black` failed. This fixes that warning.

-   [X] **The code has been formatted (`make lint-fix`)**

kensternberg-authentik committed:
core: adjust styling to meet our standards

fheisler reviewed (approved):
looks
        good
    !

kensternberg-authentik committed:
Fix version correspondence bug in website (#10278)

web: update lockfile to correspond to package.json, enabling npm-ci

Looks like someone updated `package.json` to have the latest version
of Typescript, but failed to update `package-lock.json` to get that
version into the cache.  `npm ci` won't work if the versions in the
two files don't correspond, as that means, well, exactly that: no
one has checked that the versioning is correct.

The actual diff to `package-lock.json` is small enough it can be
verified by eye, and it's fine.
